factcast-integration-full/factcast-integration-full-tls/src/test/java/org/factcast/example/tls/client/TLSClientTest.java

```diff
@@ -0,0 +1,48 @@
+package org.factcast.example.tls.client;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import java.io.File;
+
+import org.factcast.core.Fact;
+import org.factcast.core.FactCast;
+import org.factcast.core.spec.FactSpec;
+import org.factcast.core.subscription.Subscription;
+import org.factcast.core.subscription.SubscriptionRequest;
+import org.junit.ClassRule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.testcontainers.containers.DockerComposeContainer;
+import org.testcontainers.containers.wait.strategy.HostPortWaitStrategy;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+public class TLSClientTest {
+    @ClassRule
+    public static DockerComposeContainer<?> compose = new DockerComposeContainer<>(
+            new File("docker-compose.yml"))
+                    .withExposedService("db", 5432, new HostPortWaitStrategy())
+                    .withExposedService("factcast", 9443,
+                            new HostPortWaitStrategy());
+
+    @Autowired
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/uwe/factcast/pullRequest?prid=4846669)